pkg/controller/utils/conditions.go

```diff
@@ -163,6 +163,15 @@ func SetAccountCondition(
 		// Need to always update the probe time, so if the condition occurs again
 		// or we probe and the condition is still active, the date is updated.
 		existingCondition.LastProbeTime = now
+		conditions = append(conditions,
+			awsv1alpha1.AccountCondition{
+				Type:               conditionType,
+				Status:             existingCondition.Status,
+				Reason:             existingCondition.Reason,
+				Message:            existingCondition.Message,
+				LastTransitionTime: existingCondition.LastTransitionTime,
+				LastProbeTime:      existingCondition.LastProbeTime,
+			})
```

Comment:
I would remove line 165 and change line 173 to:
```
LastProbeTime:      now,
```

Reply:
@katherinelc321 can you update the code with Ari's recommendation?